index.html

```diff
@@ -62,12 +63,62 @@ <h2 class="info-title">I'am a Software Developer</h2>
               /></a>
             </li>
           </ul>
+          <div class="home-section">
+            <h1 class="section-title">Hey There. i'm Alex</h1>
+            <article class="info-container">
+              <h2 class="info-title">I'am a Software Developer</h2>
+              <p class="info">
+                I can help you build a product , feature or website Look through
+                some of my work and experience! If you like what you see and
+                have a project you need coded, don't hestiate to contact me.
+              </p>
+            </article>
+            <ul class="links-container">
+              <li>
+                <a
+                  href="http://https://github.com/ambrose-kibet"
+                  target="_blank"
+                  ><img src="./assets/git.svg"
+                /></a>
+              </li>
+              <li>
+                <a
+                  href="http://https://github.com/ambrose-kibet"
+                  target="_blank"
+                  ><img src="./assets/linkedin.svg"
+                /></a>
+              </li>
+              <li>
+                <a
+                  href="http://https://github.com/ambrose-kibet"
+                  target="_blank"
+                  ><img src="./assets/angelist.svg"
+                /></a>
+              </li>
+              <li>
+                <a
+                  href="http://https://github.com/ambrose-kibet"
+                  target="_blank"
+                  ><img src="./assets/twitter.svg"
+                /></a>
+              </li>
+              <li>
+                <a
+                  href="http://https://github.com/ambrose-kibet"
+                  target="_blank"
+                  ><img src="./assets/medium.svg"
+                /></a>
+              </li>
+            </ul>
+          </div>
         </div>
       </section>
       <section class="section-container">
```

Comment:
- [ ] Your projects section background color is a little bit different from the template. The template has a white background color. Kindly add that.

Reply:
![microverse3](https://user-images.githubusercontent.com/95437850/206190876-ec1f0fba-68a5-4eb0-820a-e5ac3d7eba89.png)
my code css :`.section-container {
  background: #e5e5e5;
  min-height: 100vh;
  display: grid;
  align-items: center;
  justify-content: center;
}` is correct as per the design